newsroom/core/resources/service.py

```diff
@@ -33,3 +34,40 @@ async def on_deleted(self, doc: NewshubResourceModelType):
         if self.clear_item_cache_on_update:
             app = get_current_wsgi_app()
             app.cache.delete(str(doc.id))
+
+    async def update_many(self, lookup: Dict[str, Any], updates: Dict[str, Any]) -> List[str]:
```

Comment:
This would not be needed anymore. Also this would be better placed in the superdesk-core as it's not Newshub specific

Reply:
I will take care of this addition